### PR TITLE
FINERACT-1241-elastic-web-hook-update

### PR DESCRIPTION
**Pull request by fynmanoj**
FINERACT-1241

## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [x] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [x] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm with details of any API changes

- [x] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Review by vorburger — CHANGES_REQUESTED** [review]
@fynmanoj nice! This will "conflict" with my #1486 from yesterday.. would you be willing to review that PR and if you are OK with it have @avikganguly01 who is a committer first merge that one for me (if it looks good to both of you) and then rebase this on it and make the required `retrofit` => `retrofit2` etc. adaptations here after #1486 is merged?

**Comment by fynmanoj** [comment]
@vorburger, sure. kindly drop me a comment here once #1486 is merged. I shall re work on this PR

**Comment by avikganguly01** [comment]
@fynmanoj : Michael's PR merged.

**Comment by vorburger** [comment]
@fynmanoj you can rework this PR now... we would love to merge it for you when conflicts are resolved and it passes the build.

File: fineract-provider/dependencies.gradle
Comment (vorburger) [inline fineract-provider/dependencies.gradle]: @fynmanoj sorry I keep being a PITA here, but this `retrofit2:converter-gson:2.1.0` here is not good - you shouldn't fix the version here - because we already fixed a version in the root `fineract/build.gradle` (not `fineract-provider/` - but there we currently have `2.9.0` instead of your old `2.1.0`.. would you be able to just drop the version number here (it should work, and get "inherited", or was there a reason why you NEED `2.1.0` specifically?

```suggestion
            'com.squareup.retrofit2:converter-gson'
```

File: fineract-provider/src/main/java/org/apache/fineract/commands/service/SynchronousCommandProcessingService.java
Comment (vorburger) [inline fineract-provider/src/main/java/org/apache/fineract/commands/service/SynchronousCommandProcessingService.java]: Just removing such lines entirely instead of leaving them commented out like this is less "cluttered" (there is always git, for history):

```suggestion
        publishEvent(wrapper.entityName(), wrapper.actionName(), command, result);
```

**Review by vorburger — CHANGES_REQUESTED** [review]
feedback re Gradle dependency needs to be addressed before we can merge

File: fineract-provider/src/main/java/org/apache/fineract/commands/service/SynchronousCommandProcessingService.java
Comment (fynmanoj) [reply fineract-provider/src/main/java/org/apache/fineract/commands/service/SynchronousCommandProcessingService.java]: corrected!

**Comment by fynmanoj** [comment]
@vorburger, the changes metioned are added

**Review by avikganguly01 — APPROVED** [review]

**Pull request merged**
